### DISCUSSION TITLE
[help] It takes more than 24 hours for 11% plot creation

### DISCUSSION THREAD

**Discussion by byEpay** [Chia-Network/chia-blockchain#3357] [Support]
He has been creating the plot for more than 24 hours and I do not know why, I have activated the option for processors with more than 10 years, everything should be fine, right?

**The last lines of the record were the following:**

Forward propagation table time: 6695.476 seconds. CPU (121.090%) Fri Apr 30 20:26:47 2021
Computing table 4
	Bucket 0 uniform sort. Ram: 3.251GiB, u_sort min: 1.625GiB, qs min: 0.812GiB.
	Bucket 1 uniform sort. Ram: 3.251GiB, u_sort min: 3.250GiB, qs min: 0.813GiB.
	Bucket 2 uniform sort. Ram: 3.251GiB, u_sort min: 1.625GiB, qs min: 0.812GiB.
	Bucket 3 uniform sort. Ram: 3.251GiB, u_sort min: 3.250GiB, qs min: 0.813GiB.

**My pc**
- Intel Core 2 Quad Q6600 2.40 GHz
- 4 GB RAM ddr3
- 4 TB HDD

**Comment by byEpay** [comment]
I have canceled the plot and put it back in and when I reach 11% I get this error:

Only read 0 of 262134 bytes at offset 0 from "T:\\plot-k32-2021-05-02-10-53-e5b4ba9c088132cb821473561919dfb112bf2331c2966b9b46f208f688b520b9.plot.p1.t2.sort_bucket_122.tmp"with length 603893862. Error 1. Retrying in five minutes.
Only read 0 of 262134 bytes at offset 0 from "T:\\plot-k32-2021-05-02-10-53-e5b4ba9c088132cb821473561919dfb112bf2331c2966b9b46f208f688b520b9.plot.p1.t2.sort_bucket_122.tmp"with length 603893862. Error 1. Retrying in five minutes.
Only read 0 of 262134 bytes at offset 0 from "T:\\plot-k32-2021-05-02-10-53-e5b4ba9c088132cb821473561919dfb112bf2331c2966b9b46f208f688b520b9.plot.p1.t2.sort_bucket_122.tmp"with length 603893862. Error 1. Retrying in five minutes.

All the time the error message, it takes 1 hour to get the same message.

**Reply by isis59** [reply]
There's no space left on your disk.
To make plotting faster, you should use SSD for your temp folder. 

As a remember, creating ONE k32 plot ( about 103 GB once done) uses about 300 GB on your temp folder.

**Reply by byEpay** [reply]
I have a 2tb hdd hard disk for temporary files, even if it is slower it should work, the disk is completely empty only for temporary files. I am creating 100gb plots, there is plenty of space for temporary 300gb

**Comment by ning-mz** [comment]
Hi there, have you solved this problem? I got the same issue, I'm using a SN550 1T SSD as the tmp drive and a 2T HDD as the plot target with k=33 and sequential queue job. Thank you.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
